source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -74,7 +75,6 @@ namespace aspect
     EllipsoidalChunk<dim>::EllipsoidalChunkGeometry::push_forward_ellipsoid(const Point<3> &phi_theta_d, const double semi_major_axis_a, const double eccentricity) const
     {
       AssertThrow (dim == 3,ExcMessage ("This can currently only be used in 3d."));
-
```

Comment:
why remove all of these empty lines? I think they help readability of the code.

Reply:
I think this comment still applies

Reply:
ah, I missed this one. Fixed it :)